lib/verk/supervisor.ex

```diff
@@ -16,10 +16,10 @@ defmodule Verk.Supervisor do
 
   @doc false
   def init(_) do
-    queues = Application.get_env(:verk, :queues, [])
+    queues = Confex.get(:verk, :queues, [])
```

Comment:
Supposedly it can work though:

```
Confex.get(:myapp, MyIntKey)
[a: 123, b: "abc"]
```
So if you give a value like MY_ENV=a:123,b:abc it should actually work